beam_jobs/base_model_validator.py

```diff
@@ -0,0 +1,239 @@
+# coding: utf-8
+#
+# Copyright 2021 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import absolute_import  # pylint: disable=import-only-modules
+from __future__ import unicode_literals  # pylint: disable=import-only-modules
+
+import datetime
+import re
+
+import apache_beam as beam
+
+from core.domain import cron_services
+from core.platform import models
+import python_utils
+
+
+
+(base_models, user_models) = models.Registry.import_models(
+    [models.NAMES.base_model, models.NAMES.user])
+datastore_services = models.Registry.import_datastore_services()
+
+
+ERROR_CATEGORY_COMMIT_CMD_CHECK = 'commit cmd check'
```

Comment:
Let's limit these to what's actually used, we'll introduce new ones as we need them.